### PR TITLE
allow to reinstall automatic updates

### PR DESCRIPTION
**Pull request by szaimen**
Signed-off-by: szaimen <szaimen@e.mail.de>

### CODE REVIEW

File: addons/automatic_updates.sh [hunk 1]
```diff
@@ -18,29 +21,30 @@ debug_mode
...
+if ! crontab -u root -l | grep -q "$SCRIPTS/update.sh"
+then
+    # Ask for installing
+    install_popup "$SCRIPT_NAME"
+else
+    # Ask for removal or reinstallation
+    reinstall_remove_menu "$SCRIPT_NAME"
+    # Removal
+    crontab -u root -l | grep -v "$SCRIPTS/update.sh"  | crontab -u root -
+    sed -i "s|/sbin/shutdown -r +1||g" "$SCRIPTS"/update.sh
```
Comment (enoch85) [inline addons/automatic_updates.sh:34]: I think we use `reboot` in an earlier version as well. Please check history.

File: addons/automatic_updates.sh [hunk 2]
```diff
@@ -18,29 +21,30 @@ debug_mode
...
-    then
-        sed -i "s|exit|/sbin/shutdown -r +1|g" "$SCRIPTS"/update.sh
-        echo "exit" >> "$SCRIPTS"/update.sh
-    fi
-    msg_box "Please remember to keep backups in case something should go wrong, you never know." "$SUBTITLE"
+    sed -i "s|exit|/sbin/shutdown -r +1|g" "$SCRIPTS"/update.sh
+    echo "exit" >> "$SCRIPTS"/update.sh
 fi
+
+msg_box "Please remember to keep backups in case something should go wrong, you never know."
```
Comment (enoch85) [inline addons/automatic_updates.sh:48]: Please change from *should* to *would*.

File: addons/automatic_updates.sh (hunk 1 above)
Comment (szaimen) [reply addons/automatic_updates.sh]: hm... I think I am just deleting this line... done

File: addons/automatic_updates.sh (hunk 2 above)
Comment (szaimen) [reply addons/automatic_updates.sh]: I've deleted the word since it sounds wrong

File: addons/automatic_updates.sh (hunk 1 above)
Comment (enoch85) [reply addons/automatic_updates.sh]: No, we don't want to reboot if it's uninstalled... Don't make it easy, make it stable. :D

**Comment by szaimen** [comment]
Is anything missing here?

File: addons/automatic_updates.sh (hunk 1 above)
Comment (szaimen) [reply addons/automatic_updates.sh]: done

**Comment by szaimen** [comment]
This should be done now

**Comment by szaimen** [comment]
@enoch85 is anything missing here?

**Comment by enoch85** [comment]
> @enoch85 is anything missing here?

No, is just don't have time. Need to test this first. :+1:

**Comment by enoch85** [comment]
`AUT_UPDATES_TIME` does not work.

Please have a look.

**Review by enoch85 — CHANGES_REQUESTED** [review]
See comment

**Comment by szaimen** [comment]
> `AUT_UPDATES_TIME` does not work.

I`ve just tested this and it works for me.

Maybe there is something wrong with your test instance?

**Comment by enoch85** [comment]
It's a production server. ;)

Please check again, the script explainer doesn't pick up the variable.

**Comment by szaimen** [comment]
Ah, this is the problem. Thanks for clarifying!

**Comment by szaimen** [comment]
Should work now 👍

**Pull request merged**